packages/scheduler/lib/workers/cleanup/cleanup.worker.ts

```diff
@@ -0,0 +1,112 @@
+import * as fs from 'fs';
```

Comment:
Don't know enough about worker but I feel like it's not going to help.
It creates a whole new nodejs instance just to run a setInterval, locally I can definitely the dev is slower than before I just to be mindful about memory and cpu consumption

Reply:
Also discussed online. It is true that worker_threads add overheard but in prod we are very far from maxing out the orchestrator instance resources even though we are running on very small vms. It might also have an impact for local setup but I will prefer to go for this solution and only revisit if it becomes unbearably slow for local development

Reply:
Got it. Would be sure to test and verify this on staging to make sure no spikes occur with a lot of schedules. Happy to help generate more staging schedules if needed.

Reply:
as mentioned below, we are gonna delete one schedule at a time to minimize as much as we can the impact of schedule deletions